apps/passport-server/src/services/generic-issuance/pipelines/LemonadePipeline.ts

```diff
@@ -913,25 +950,32 @@ export class LemonadePipeline implements BasePipeline {
     // ineffective.
     delete ticketCopy.timestampSigned;
     const hash = await getHash(
-      JSON.stringify(ticketCopy) + eddsaPrivateKey + pipelineId
+      JSON.stringify(ticketCopy) + eddsaPrivateKey + pipelineId + ticketPCDType
```

Comment:
this may be a problem w/ the existing cache but we should probably use `stableStringify` here rather than `JSON.stringify`